internal/integrations/github/github.go

```diff
@@ -0,0 +1,47 @@
+package github
+
+import (
+	"context"
+	"errors"
+
+	"github.com/google/go-github/v26/github"
+	"golang.org/x/oauth2"
+)
+
+// GitHub handles integrations with the GitHub API
+type GitHub struct {
+	Org    string
+	Client *github.Client
+}
+
+// New creates a new GitHub integration
+func New(org string, accessToken string) (*GitHub, error) {
+	ctx := context.Background()
+	ts := oauth2.StaticTokenSource(&oauth2.Token{AccessToken: accessToken})
+	tc := oauth2.NewClient(ctx, ts)
+	client := github.NewClient(tc)
+
+	return &GitHub{
+		Client: client,
+		Org:    org,
+	}, nil
+}
+
+// GetTravisCIBuildURLForRef uses the Checks API to find the URL to the Travis build for the specified ref
+func (github *GitHub) GetTravisCIBuildURLForRef(repo string, ref string) (string, error) {
```

Comment:
The `github` method receiver shadows the `go-github/github` package symbol. Usually method receiver vars are abbreviated because their scope is limited and the context they're used in is well understood. Should we use an abbreviated method receiver here?